pkgs/applications/version-management/git-together/default.nix

```
@@ -0,0 +1,31 @@
+{ pkgs, lib, stdenv, fetchFromGitHub, rustPlatform, openssl, Security }:
```

```suggestion
{ lib
, stdenv
, fetchFromGitHub
, rustPlatform
, openssl
, Security }:
```
This is the format usually used in nixpkgs.

---

Will do! I've been using `nixfmt` on save. Is there a different nixfmt config I can use to match nixpkgs?

---

> Will do! I've been using `nixfmt` on save. Is there a different nixfmt config I can use to match nixpkgs?

I'm not sure, need to search about.